lib/beaker/cli.rb

```diff
@@ -113,6 +113,15 @@ def execute!
         end
       #cleanup phase
       rescue => e
+        begin
+          if @options[:fail_mode].to_s =~ /slow/
```

Comment:
@highb yeah, I think you're right, that we should keep with the definition that it should always run, so it shouldn't check for slow mode like the post suite does. Other than that, everything looks good here. Once that change is through, we'll run it through one last automation pass & we should be good to merge this.